corehq/apps/users/forms.py

```diff
@@ -961,14 +961,16 @@ def __init__(self, group_api_url, *args, **kwargs):
         super(GroupMembershipForm, self).__init__(*args, **kwargs)
         self.fields['selected_ids'].widget.set_url(group_api_url)
 
-        self.helper = FormHelper()
-        self.helper.label_class = 'form-label'
+        self.helper = HQFormHelper()
         self.helper.form_tag = False
+        self.helper.form_class = 'form-horizontal'
```

Comment:
nit: I think this line is not needed since `HQFormHelper` uses that class.

Reply:
indeed, thanks - 0814e07e017